azure-pipelines-product-construction-service.yml

```diff
@@ -0,0 +1,67 @@
+# Changes the format of the Build.BuildNumber variable
+name: $(Date:yyyyMMdd)$(Rev:r)
+
+trigger:
+  branches:
+    exclude:
+      - production
+
+variables:
+- name: resourceGroupName
+  value: 'product-construction-service'
+- name: containerName
+  value: 'product-construction-service.api'
+- ${{ if ne(variables['Build.SourceBranch'], 'refs/heads/production') }}:
+  - group: 'Product-Construction-Service-Int'
+  - name: containerappName
+    value: product-construction-int
+  - name: containerRegistryName
+    value: productconstructionint
+  - name: subscriptionName
+    value: HelixStaging
+  - name: containerappEnvironmentName
+    value: product-construction-service-env-int
+  - name: dockerRegistryUrl
+    value: productconstructionint.azurecr.io
+
+stages:
+- stage: PublishProductConstructionService
+  displayName: Publish Product Construction Service
+  dependsOn: []
+  jobs:
+  - job: Publish
+    displayName: Publish Product Construction Service
+    pool:
+      vmImage: ubuntu-latest
+      # name: NetCore1ESPool-Internal
+      # demands: ImageOverride -equals 1es-ubuntu-2004
```

Comment:
We build the rest of the repo on Windows only, we use Windows locally too. Why Ubuntu?

Reply:
I wasn't able to change the Docker daemon to Linux on the `windows-latest` image I originally tried using. I'll try some other image, it might be configured to Linux by default

Reply:
Ah, nevermind then